experimental/PartitionedPermutations/docs/src/introduction.md

```diff
@@ -0,0 +1,37 @@
+```@meta
+CurrentModule = Oscar
+```
+
+# Partitioned Permutations
+
+Partitioned Permutations are used in the context of Free Probability Theory to model higher order freeness and higher order free cumulants, see e.g. [CMS07](@cite).
+
+We provide basic functions for working with partitioned permutations. The focus is on factorizing partitioned permutations.
+
+# Basics
+
+Formally, a partitioned permutation $(V, \pi)$ consists of a permutation $\pi$ and a partition $V$ of the set $\{1, ..., n\}$ such that the partition dominates the permutation in the sense that every cycle of $\pi$ is contained in one block of $V$. We call $n$ the length of $(V, \pi)$. Mathematically, another length function is more important. It is given by
+$$|(V, \pi)| := n - ( 2 \cdot \text{number of blocks of } V - \text{number of cycles of } \pi),$$
+and we call this the adjusted length of $(V, \pi)$. Note that this terminology is not used in the literature.
+
+Technically, we rely on the module `SetPartitions` for working with partitions.
```

Comment:
Hm, there is no module `SetPartitions` in the Julia sense, so this might lead to confusion. Maybe leave this line out -- this is anyway just about the internals of the code.